src/build/prepareElectronApp.ts

```diff
@@ -111,41 +112,33 @@ async function maybeCopyScripts(srcs: string[], dest: string): Promise<void> {
       );
     }
 
-    let destFileName: string;
-    if (path.extname(src) === '.js') {
-      destFileName = 'inject.js';
-    } else if (path.extname(src) === '.css') {
-      destFileName = 'inject.css';
-    } else {
-      return;
+    if (supportedInjectionExtensions.indexOf(path.extname(src)) < 0) {
+      console.log(`Skipping unsupported injection file: ${src}`);
+      continue;
     }
 
+    const postFixHash = generateRandomSuffix();
+    const destFileName = `inject-${postFixHash}.${path.extname(src)}`;
     const destPath = path.join(dest, 'inject', destFileName);
     log.debug(`Copying injection file "${src}" to "${destPath}"`);
     await copyFileOrDir(src, destPath);
   }
 }
 
-function normalizeAppName(appName: string, url: string): string {
-  // use a simple 3 byte random string to prevent collision
-  const hash = crypto.createHash('md5');
-  hash.update(url);
-  const postFixHash = hash.digest('hex').substring(0, 6);
+function normalizeAppName(appName: string): string {
+  // use a simple random string to prevent collision
+  const postFixHash = generateRandomSuffix();
```

Comment:
By the way, I see you commented on an issue about this hash/suffix. This is related to https://github.com/nativefier/nativefier/pull/888 . Is the reason for this hash clear to you, and do you think we should keep doing it?

Reply:
It is clear to me, and I do think we should keep doing it. I do not see the harm in it as long as we reference the app name correctly (i.e., user provided/inferred) where appropriate instead of this internal one.